### ISSUE TITLE
How to create pipelines from AutoSKlearn output configurations?

### ISSUE THREAD

**Issue by alinedelvalle** [automl/auto-sklearn#1752]
Hello,
I use AutoSKlearn to find a pipeline with a multi-label classifier (ensemble size is 1). I have the pipeline configuration as the response. I would like to use this configuration to train pipelines with different training and testing sets. I used the fit_pipeline method passing the training and testing data, but regardless of the data I pass, the results are always the same. Is it correct to use fit_pipeline? If so, what could be the error?

**Comment by simonprovost** [comment]
Hi there!

I am far from maintaining Auto-Sklearn nor part of the Auto-Sklearn team, but I was passing by and hope this might help.

I believe you are looking for `refit` following the discovery of your champion model/ensemble: https://automl.github.io/auto-sklearn/master/api.html?highlight=refit#autosklearn.classification.AutoSklearnClassifier.refit

Hope this helps,

Cheers
